### ISSUE TITLE
How to use IceVision without using from icevision.all import *

### ISSUE THREAD

**Issue by AlexandreBrown** [airctic/icevision#1116]
## 📓 Documentation Update
**What part of documentation was unclear or wrong?**
To me, it is unclear which imports are needed in order to use ice vision.  
For instance, what is the import that is needed for the various models?  

**Describe the solution you'd like**
I'd like a clear description and examples of how to get started without using a wildcard import (`from icevision.all import *`).  

**Additional context**
In our case, we cannot use a wildcard import because our code is running as part of a [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/sdk-v2/v2-compatibility/).  
Long story short, it means our code is being executed inside a function, this function is then deployed to a docker image and executed. There is no such thing as a top level import for us.     
This means that we must use imports inside our main function, eg:  
![Screenshot from 2022-07-04 14-55-47](https://user-images.githubusercontent.com/26939775/177206257-fc3b4a68-d654-45f2-8e5d-2f2b654dbf73.png)  
  
The following produces an error : `SyntaxError: import * only allowed at module level`

**Comment by AlexandreBrown** [comment]
Potential solution for the doc :  
Mention that if it is not possible to use wildcard import `*` , then that you can use `import icevision.all as iv`.  
Then instead of `models.mmdet.retinanet`, you would do `iv.models.mmdet.retinanet`  
  
I will make a doc PR if my tests are successful.

**Comment by AlexandreBrown** [comment]
I can confirm that using `import icevision.all as iv` works as expected, I will make a doc PR next weekend about it.
